Makefile

```diff
@@ -42,12 +42,13 @@ prepush: go_build go_test lint
 python_test: python3 tox
 	cd $(WPTD_PATH)results-processor; tox
 
-go_build: git mockgen $(WPTD_GO_PATH)
-	cd $(WPTD_GO_PATH); go get -v ./...
-	cd $(WPTD_GO_PATH); go generate ./...
+go_build: git mockgen packr
+	cd $(WPTD_PATH); go get -v ./...
+	cd $(WPTD_PATH); go generate ./...
+	cd $(WPTD_PATH); go build ./...
 
 go_build_test: go_build gcc
-	cd $(WPTD_GO_PATH); go get -v -t -tags="small medium large" ./...
```

Comment:
I tried this locally, outside of Docker, with Go 1.13 and the `-t` flag was needed to get the test packages.

I dug a bit deeper and found this annoying back-and-forth:
* Modules were formally introduced in Go 1.11 along with a module-aware mode of `go get` in which `-t` was a no-op: https://github.com/golang/go/blob/83bd88129ea3b5a1272c06db3e865ee75740cee8/src/cmd/go/alldocs.go#L2445
* Go devs reverted the change in 1.13 to make `go get` behave more consistently with and without modules and `-t` is required again: https://golang.org/doc/go1.13#modules:
> By default, go get -u in module mode upgrades only non-test dependencies, as in GOPATH mode. It now also accepts the -t flag, which (as in GOPATH mode) causes go get to include the packages imported by tests of the packages named on the command line.

Reply:
Welp. Good to know, I guess.

Reply:
Actually, this doesn't matter because we don't want to upgrade the packages in these targets.

We can have a new make target for updating everything including test packages.